tests/percy/multilangphotographer.js

```diff
@@ -0,0 +1,170 @@
+/**
+ * @typedef {import('./pagenavigator.js')} PageNavigator
+ * @typedef {import('./Camera.js')} Camera
+ */
+
+/**
+ * Responsible for determining which snapshots to take for specific locale
+ */
+class MultilangPhotographer {
+  /**
+   * @param {PageNavigator} pageNavigator
+   * @param {Camera} camera
+   * @param {locale} locale locale set for the site when taking snapshot
+   */
+  constructor(pageNavigator, camera, locale) {
+    this._pageNavigator = pageNavigator;
+    this._camera = camera;
+    this._locale = locale;
+
+    /**
+     * Custom queries for specific locale
+     * 
+     * @type {Object<string, string>}
+     */
+    this._queries = {};
+
+    /**
+     * Locale tag to add to snapshot name
+     * 
+     * @type {string}
+     */
+    this.localeSnapshotTag = '';
+
+    this.setLocale(locale);
+  }
+
+  /**
+   * Sets locale for the page navigation and snapshot tag
+   * 
+   * @param {string} locale locale of the site
+   */
+  setLocale(locale) {
+    this._locale = locale;
+    this.localeSnapshotTag = this._locale === 'en'? '' : this._locale + '--';
+    this._setLocaleQueries();
+  }
+
+  /**
+   * Sets custom queries based on locale
+   */
+  _setLocaleQueries() {
```

Comment:
It seems like now the multilang phtographer has two repsonsibilites: defining the test suite for multilang tests and defining the multilang translations to be used in the test suit. What do you think about instead passing in the translated queries to the MultilangPhotographer as an object rather than defining them here? We could have a class or a json file that contains the translations for each locale, and then we load the translations for the specific locales and pass it to the multilang photographer

Reply:
I like the idea of using a json file to hold the translated queries. If we do pass the translation of specific locale to `multilangphotographer` (and have camera handles the addition locale name tag), then the photographer no longer need to have knowledge of locale. Although, I'm thinking we could still pass locale in multilangphotographer (along with the queries) and have it update locale property for its navigator and camera as well rather than updating each component separately in index.js. What do you think?

Reply:
Either way sounds good to me